Dockerfile.doc

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
```

Comment:
The default for the original `Dockerfile` is `debian:bullseye`. Is there a reason to use `ubuntu:22.04` instead?

Reply:
Fair point - we target Debian so we should use it also for documentation.